test/CMakeLists.txt

```diff
@@ -79,6 +79,10 @@ function(add_performance_test TEST_NAME COMM RMW_IMPLEMENTATION SYNC_MODE)
       set(SKIP_TEST "SKIP_TEST")
     endif()
 
+    if(PERF_TEST_SKIP)
```

Comment:
should we define the default value?

Reply:
Sure, we can. It defaults to undefined, which evaluates to false. I'll add it anyway.

Reply:
7cefc21